src/command_modules/azure-cli-sql/azure/cli/command_modules/sql/custom.py

```diff
@@ -812,11 +818,31 @@ def elastic_pool_list_capabilities(
 #                sql server                   #
 ###############################################
 
+def server_create(
+        client,
+        resource_group_name,
+        server_name,
+        assign_identity=False,
+        **kwargs):
+
+    if assign_identity is True:
+        kwargs['identity'] = ResourceIdentity(type="SystemAssigned")
+
+    # Create
+    return client.create_or_update(
+        server_name=server_name,
+        resource_group_name=resource_group_name,
+        parameters=kwargs)
+
 
 # Update server. Custom update function to apply parameters to instance.
 def server_update(
         instance,
-        administrator_login_password=None):
+        administrator_login_password=None,
+        assign_identity=False):
+
+    if instance.identity is None and assign_identity is True:
+        instance.identity = ResourceIdentity(type="SystemAssigned")
```

Comment:
same comments as above

Reply:
Can you comment that identity cannot be removed? Otherwise this code looks suspicious.